### PR TITLE
Minor map fixes from reset

### PR DESCRIPTION
**Pull request by FlamingLily**
## About The Pull Request
Fixes missing things

## Why It's Good For The Game
Fix

## Changelog
:cl:
fix: Stasis units exist again
fix: 😳 machines exist again
fix: Holobadge boxes exist again
fix: Gas Miners exist again (and atmos now uses sexy decals like Journey instead of canisters to indicate their contents)
/:cl:

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation_skyrat.dmm [hunk 1]
```diff
@@ -24083,7 +24039,7 @@
 /obj/effect/decal/cleanable/dirt/dust,
 /obj/structure/closet/cardboard,
 /obj/effect/spawner/lootdrop/maintenance/two,
-/obj/item/toy/plush/lizard_plushie/green{
+/obj/item/toy/plush/lizardplushie{
```
Comment (Funce) [inline _maps/map_files/MetaStation/MetaStation_skyrat.dmm]: Mm, I changed that a couple of days ago

**Review by Funce — COMMENTED** [review]
I think you may have an old version of the map you're merging in here. Couple of these are reverting some changes I remember.

File: _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm [hunk 2]
```diff
@@ -50,7 +50,7 @@
 /obj/item/clothing/head/chefhat,
 /obj/effect/turf_decal/bot,
 /obj/machinery/button/door/directional/south{
-	id = "kitchenwindows";
+	id = "kitchenwindow";
```
[the remark this answers is not in the document]
Comment (FlamingLily) [reply _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm]: nooo, i dont even know why that happened especially considering i merged to master

File: _maps/map_files/MetaStation/MetaStation_skyrat.dmm (hunk 1 above)
Comment (FlamingLily) [reply _maps/map_files/MetaStation/MetaStation_skyrat.dmm]: yeah that one i dont know why it  happened either

File: _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm (hunk 2 above)
[the remark this answers is not in the document]
Comment (FlamingLily) [reply _maps/map_files/Deltastation/DeltaStation2_skyrat.dmm]: it...

it's "kitchenwindows" on my sdmm


why

**Review by Funce — APPROVED** [review]
@KathrinBailey Looks all fine to me. I'll leave you to merge this considering maps get wacky.

**Comment by KathrinBailey** [comment]
small issue with delta because I'm not sure if anyone has attempted to reset it yet or what but I'm half way through doing so, I just don't have time.
literally get home from work, shower and eat, fall asleep exhausted, then wake up and it's pretty much 30m until I gotta work, so unfortunately I'll take longer than usual to get around.

**Comment by FlamingLily** [comment]
> small issue with delta because I'm not sure if anyone has attempted to reset it yet or what but I'm half way through doing so, I just don't have time.
> literally get home from work, shower and eat, fall asleep exhausted, then wake up and it's pretty much 30m until I gotta work, so unfortunately I'll take longer than usual to get around.

Take your time ❤️

**Pull request merged**